ethexe/db/src/database.rs

```diff
@@ -444,6 +447,60 @@ impl Database {
         self.cas.write(data)
     }
 
+    pub fn get_offchain_transaction(&self, tx_hash: H256) -> Option<SignedOffchainTransaction> {
+        self.kv
+            .get(&KeyPrefix::Transaction.one(tx_hash))
```

Comment:
maybe specify in keyPrefix that it's "Signed" type to avoid misuse?

Reply:
done